lib/interfaces/bdd.js

```diff
@@ -112,3 +112,5 @@ module.exports = function bddInterface(suite) {
     };
   });
 };
+
+module.exports.description = '[default] BDD or RSpec style';
```

Comment:
`module.exports.description = 'BDD or RSpec style [default]';`

Reply:
heh, I had it that way then changed it...

- [ ] revert